sdk/storage/storage-blob/src/BlockBlobClient.ts

```diff
@@ -1098,15 +1099,15 @@ export class BlockBlobClient extends BlobClient {
    * @param {Readable} stream Node.js Readable stream
    * @param {BlockBlobClient} blockBlobClient A BlockBlobClient instance
    * @param {number} bufferSize Size of every buffer allocated, also the block size in the uploaded block blob
```

Comment:
>@param {number} bufferSize Size of every buffer allocated, also the block size in the uploaded block blob [](start = 5, length = 105)

Can we describe the default value for these 2 parameter descriptions?

Reply:
Sure!